src/Box.java

```diff
@@ -0,0 +1,4 @@
+public abstract class Box extends Mail{
+    protected double weight;
```

Comment:
would be nicer to have a constructor here, so we force all children of this class to give a value to weight. this way we ensure that its not empty and ready to use